src/instructlab/eval/mmlu.py

```diff
@@ -158,6 +159,9 @@ def _simple_evaluate_with_error_handling(self, **kwargs):
                 ose
             ) or "does not appear to have a file named" in str(ose):
                 raise ModelNotFoundError(self.model_path) from ose
+            if "is not a valid JSON file" in str(ose):
+                reason = "Looked for valid JSON file but couldn't find one - are you pointing at a safetensors directory with a 'config.json'?"
+                raise InvalidModelError(self.model_path, reason) from ose
```

Comment:
what happens if someone passes in a directory that contains a gguf file and a valid `config.json` file? 
maybe we need something similar to how `is_model_gguf` is implemented in the CLI?

Reply:
I can just remove `safetensors` from the message